stock_pull_list/views/stock_picking_views.xml

```diff
@@ -0,0 +1,22 @@
+<odoo>
+    <record model="ir.actions.server" id="generate_pull_list">
+        <field name="name">Generate Pull List</field>
+        <field name="model_id" ref="stock.model_stock_picking" />
+        <field name="binding_model_id" ref="stock.model_stock_picking" />
+        <field name="state">code</field>
+        <field name="code">
+            action = records.action_create_pull_list()
+        </field>
+    </record>
```

Comment:
may be is better to move on a new `data/data.xml`

Reply:
https://github.com/odoo/odoo/blob/d153cfcdbcae0148c58aed0e4d6bc19bced858af/addons/account/views/account_account_views.xml#L151
I don't think there are differences, since I have seen both variants

Reply:
from guidelines:


https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#complete-structure

naming convention:

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#security-view-and-action

Reply:
In both links, I don't see exactly what action should be in data folder, but I understand that I need to change id of action

Reply:
Done!